src/views/profile/ProfileDetail.vue

```diff
@@ -0,0 +1,159 @@
+<script setup lang="ts">
+import { DEFAULT_NETWORK_CONFIG } from "@/helpers/config";
+import { onMounted, ref, watch } from "vue";
+import { isAddress } from "ethers/lib/utils";
+import useErc725 from "@/compositions/useErc725";
+import { useRoute, RouteLocationNormalizedLoaded } from "vue-router";
+
+const route = useRoute();
+
+const routeData = ref<RouteLocationNormalizedLoaded>(route);
+const dataSource = ref("");
+const loading = ref(true);
+const profileData = ref(null as any);
```

Comment:
Please avoid `any` when returned type is known, use `LSP3Profile` from `@lukso/lsp-factory.js` instead

```diff
- const profileData = ref(null as any);
+ const profileData = ref<{ LSP3Profile: LSP3Profile }>();
```